README.md

```diff
@@ -99,155 +99,135 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
- - Improved accessibility by supporting picking a color using the keyboard.
+ - Fixed the opaque background corners in the picker that were introduced after the upgrade to WPFUI.
 
-### Command Not Found
+### Developer Files Preview (Monaco)
 
- - Upgraded the Command Not Found to use the new PowerShell Gallery release and support ARM64. Thanks [@carlos-zamora](https://github.com/carlos-zamora)!
+ - Improved the syntax highlight for .gitignore files. Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Checking for the sticky scroll option in code behind was being done twice. Removed one of the checks. Thanks [@downarowiczd](https://github.com/downarowiczd)!
 
 ### Environment Variables Editor
 
- - Refactored, packaged and released the main Environment Variables Editor functionality as a control to be integrated into DevHome. Thanks [@dabhattimsft](https://github.com/dabhattimsft) for validating and integrating into DevHome!
-
-### FancyZones
-
- - Fixed window wrap around behavior when overriding Windows key and arrow shortcuts on single monitor scenarios. Thanks [@DanRosenberry](https://github.com/DanRosenberry)!
- - Improved accessibility of the editor by listing the keyboard shortcuts in the Canvas Editor.
+ - Added clarity to the UI section tooltips. Thanks [@anson-poon](https://github.com/anson-poon)!
 
 ### File Explorer add-ons
 
- - Updated Monaco to 0.47 and added the new sticky scroll setting for DevFiles viewer. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Added the new font size setting for DevFiles viewer. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Added support for .srt (subtitle) file previewing in DevFiles viewer. Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Fixed a crash when the preview handlers received a 64-bit handle from the OS. Thanks [@z4pf1sh](https://github.com/z4pf1sh)!
+ - Fixed a crash when trying to update window bounds and File Explorer already disposed the preview.
 
-### Hosts File Editor
+### Find My Mouse
 
- - Refactored, packaged and released the main Hosts File Editor functionality as a control to be integrated into DevHome. Thanks [@dabhattimsft](https://github.com/dabhattimsft) for validating and integrating into DevHome!
+ - Added the option to have to use the Windows + Control keys to activate. Thanks [@Gentoli](https://github.com/Gentoli)!
 
-### Image Resizer
+### Hosts File Editor
 
- - Supported narrator announcing the checkboxes in the UI and the sizes combobox. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Improved accessibility by increasing contrast in the text color of combobox items.
+ - Improved spacing definitions in the UI so that hosts name are not hidden when resizing and icons are well aligned. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Added horizontal scroll instead of wrapping in the additional lines dialog. Thanks [@htcfreek](https://github.com/htcfreek)!
```

Comment:
Nit: this is not an addition, but a **change**

Reply:
I've changed the text, thank you!